Simplified/Book/UI/NYPLBookDetailView.m

```diff
@@ -594,6 +604,9 @@ - (void)setState:(NYPLBookState)state
       self.buttonsView.hidden = NO;
       self.normalView.state = NYPLBookButtonsStateDownloadSuccessful;
       self.buttonsView.state = NYPLBookButtonsStateDownloadSuccessful;
+      if (self.downloadStarted) {
```

Comment:
why do we need to check if the download started if we are inside the `NYPLBookStateDownloadSuccessful` case?

Reply:
We only want to do the announcement if the download started and completed. Without this check, we would be doing the announcement every time user goes into the book detail page of a downloaded book.

Reply:
ahh good point!